src/app/lib/device/airplay.js

```diff
@@ -1,53 +1,80 @@
 (function (App) {
     'use strict';
 
-    var browser = require('airplay-js').createBrowser(),
+    var Bonjour = require('bonjour'),
+        AirPlayProtocol = require('airplay-protocol');
+
+    var bonjourBrowser = new Bonjour(),
         collection = App.Device.Collection;
 
-    var makeID = function (baseID) {
-        return 'airplay-' + baseID.replace(':', '');
+    var makeID = function (bonjourService) {
+        // strong ID since there can only be 1 bonjour service per host+port
+        var uniqueDeviceId = bonjourService.host + '_' + bonjourService.port;
+        return 'airplay-' + Common.md5(uniqueDeviceId);
```

Comment:
@xaiki Are you sure? I followed the Common.md5() conventions of the chromecast lib, which gives nice looking IDs: https://github.com/SteveJobzniak/butter-desktop/blob/5b0622661d98a782dd4aa9a447b251253d84441e/src/app/lib/device/chromecast.js#L14

I don't think we need the internal device ID to be "airplay-AppleTV.local:4834" or whatever, since we never look at that. If there's a problem with a device (for debugging), we've got all of that info available to us in the other fields of the AirPlay device object. The exampleDevice.service.name and exampleDevice.service.port fields are always available for the full details.